pandas/tests/indexing/multiindex/test_sorted.py

```diff
@@ -28,7 +29,8 @@ def test_getitem_slice_not_sorted(self, multiindex_dataframe_random_data):
         expected = df.reindex(columns=df.columns[:3])
         tm.assert_frame_equal(result, expected)
 
-    def test_frame_getitem_not_sorted2(self):
+    @pytest.mark.parametrize("key", [None, lambda x: x])
```

Comment:
can you move the fixture in `pandas/tests/frame/test_sorting.py` into conftest.py and re-use? (may require a more descriptive name to avoid clashes.)

Reply:
renamed the fixture `test_key` and moved to `conftest.py`.